src/new-components/text-area/text-area.stories.tsx

```diff
@@ -112,3 +114,28 @@ export function MessageToneStory() {
         </Stack>
     )
 }
+
+export function TextAreaWithoutLabelStory() {
```

Comment:
Can we duplicate this story for other fields like SelectField and TextField?

Reply:
Ok